rsksmart/rif-marketplace-ui
# feat: add link to RNS Manager

* Adds link to RNS Manager after you Buy a domain.
* RNS Manager URL has to be defined in the `ui-config.json` file.
* Checks that the URL is valid otherwise disables the button.
* Postpones Cleanup of `currentOrder` to the Transaction confirmation page.

Note: To test this PR you need to:
1) Run the RNS Manager locally with the LATEST `develop` branch.
2) If you get an error when doing `npm start` in the RNS manager run: `npm i webpack@4.28.3` in the Manager project this should fix the issue. Then try again.
3) Setup the correct RNS manager URL in the `ui-config.json` file of the MArketplace UI project. (samples provided in the default file).

4) **BUY** a domain and then click on the `Admin my domain` button of the confirmation page. You should get a new tab opened directly on the manager page (if you didnt switch the account in between, you have to be with the samne account that just bought the domain)
